### PR TITLE
add metrics acceptance question info

### PR DESCRIPTION
**Pull request by victor-schumacher**
**- What I did**
added metrics acceptance question info

**- How to verify it**
Check metabase or log the struct

**- Description for the changelog**
added metrics acceptance question info

### CODE REVIEW

**Review by lucasdittrichzup — DISMISSED** [review]

File: pkg/cmd/init.go [hunk 1]
```diff
@@ -232,6 +233,14 @@ You can view our Privacy Policy (http://insights.zup.com.br/politica-privacidade
 	responseToWrite := "yes"
 	if choose == DoNotAcceptMetrics {
 		responseToWrite = "no"
+		metricManager := metric.NewHttpSender(metric.ServerRestURL, http.DefaultClient)
```
Comment (kaduartur) [inline pkg/cmd/init.go:236]: could you receive MetricManager in your constructor?

**Review by kaduartur — CHANGES_REQUESTED** [review]

File: pkg/cmd/init.go (hunk 1 above)
Comment (victor-schumacher) [reply pkg/cmd/init.go]: yes! Done in 9e9049c15e460fe853a266f2a6be23f73a5456a2.

File: pkg/cmd/init.go [hunk 2]
```diff
@@ -232,6 +243,13 @@ You can view our Privacy Policy (http://insights.zup.com.br/politica-privacidade
 	responseToWrite := "yes"
 	if choose == DoNotAcceptMetrics {
 		responseToWrite = "no"
+		in.metricSender.Send(metric.APIData{
+			Id: "rit_init",
```
Comment (kaduartur) [inline pkg/cmd/init.go:247]: you must create a new Id for this metric because ** rit_init ** will be sent when the rit init command ends

**Review by kaduartur — CHANGES_REQUESTED** [review]

**Review by brunasilvazup — APPROVED** [review]

**Review by kaduartur — APPROVED** [review]
LGTM!

**Pull request merged**